package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@devtron-labs/devtron-fe-common-lib",
-    "version": "0.0.33",
+    "version": "0.0.34",
```

Comment:
If it is not published then change in to `"version": "0.0.33-beta-{x}"`. We increase the number when we merge it to main.